src/libsyntax/feature_gate.rs

```diff
@@ -455,6 +455,9 @@ declare_features! (
 
     // Parentheses in patterns
     (active, pattern_parentheses, "1.26.0", None, None),
+
+    // Raw identifiers allowing items with keyword names
```

Comment:
allowing keyword names to be used in arbitrary identifier positions, not only item names